.editorconfig

```
@@ -18,3 +18,7 @@ indent_size = 2
 [coderd/database/dump.sql]
 indent_style = space
 indent_size = 4
+
+[coderd/database/queries/*.sql]
```

```suggestion
[*.sql]
```

it'd be nice if everything was included. we could reformat migrations, or at least format them going forward. with all of the views we have that need to be recreated when you edit an underlying table stuff ends up getting copied and pasted around a lot.